### ISSUE TITLE
Traceback leoColorizer.py and pygments\lexer.py

### ISSUE THREAD

**Issue by LewisNeal** [leo-editor/leo-editor#3988]
Running Leo 6.8.1-devel, build eb4ce4f641
Python 3.12.4, PyQt version 6.7.2
Windows 10 AMD64 (build 10.0.19045) SP0

This is definitely a Leo problem, as the reply to [pygments issue #2753](https://github.com/pygments/pygments/issues/2753) makes clear.

When searching with Find/F3 I received this traceback:
```console
Uncaught exception in Leo...
Traceback (most recent call last):
  File "N:\git\leo-editor\leo\core\leoColorizer.py", line 2823, in highlightBlock
    self.colorizer.recolor(s)  # Highlight just one line.
    ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "N:\git\leo-editor\leo\core\leoColorizer.py", line 3243, in recolor
    self.mainLoop(s)
  File "N:\git\leo-editor\leo\core\leoColorizer.py", line 3104, in mainLoop
    for token, text in lexer.get_tokens(s):
  File "C:\Users\lewisneal\AppData\Local\Programs\Python\Python3\Lib\site-packages\pygments\lexer.py", line 270, in streamer
    for _, t, v in self.get_tokens_unprocessed(text):
  File "C:\Users\lewisneal\AppData\Local\Programs\Python\Python3\Lib\site-packages\pygments\lexer.py", line 804, in get_tokens_unprocessed
    yield from action(self, m, ctx)
               ^^^^^^^^^^^^^^^^^^^^
TypeError: PygmentsColorizer.at_language_callback() takes 3 positional arguments but 4 were given

_get_format_from_style KeyError(Token.Punctuation.Indicator)
_get_format_from_style KeyError(Token.Literal.Scalar.Plain)
```

**Comment by edreamleo** [comment]
@LewisNeal What language was Leo trying to color when pygments crashed?

**Comment by edreamleo** [comment]
Closed via PR #3999.
